src/seals/atari.py

```diff
@@ -1,15 +1,48 @@
 """Adaptation of Atari environments for specification learning algorithms."""
 
-from typing import Iterable
+from typing import Dict, Iterable, List, Optional, Tuple
 
 import gym
 
-from seals.util import AutoResetWrapper, get_gym_max_episode_steps
+from seals.util import AutoResetWrapper, MaskScoreWrapper, get_gym_max_episode_steps
 
+SCORE_REGIONS: Dict[str, List[Dict[str, Tuple[int, int]]]] = {
```

Comment:
You use `List[Dict[str, Tuple[int, int]]]` in three places in your code -- consider defining it as a type? Like:

```python
MaskedRegionSpecifier = List[Dict[str, Tuple[int, int]]]
```

I'd also consider using a named tuple instead of `dict` to enforce that `x` and `y` are both present.